frontend/src/scenes/products/Products.tsx

```diff
@@ -20,61 +23,18 @@ export const scene: SceneExport = {
     component: Products,
 }
 
-function OnboardingCompletedButton({
-    productUrl,
-    onboardingUrl,
-    getStartedActionOverride,
-}: {
-    productUrl: string
-    onboardingUrl: string
-    productKey: ProductKey
-    getStartedActionOverride?: () => void
-}): JSX.Element {
-    return (
-        <>
-            <LemonButton type="secondary" to={productUrl}>
-                Go to product
-            </LemonButton>
-            <LemonButton
-                type="tertiary"
-                onClick={() => {
-                    if (getStartedActionOverride) {
-                        getStartedActionOverride()
-                    }
-                    router.actions.push(onboardingUrl)
-                }}
-            >
-                Set up again
-            </LemonButton>
-        </>
-    )
+const ProductNameToColor = {
+    'Product analytics': 'blue',
+    'Session replay': 'var(--warning)',
+    'Feature flags & A/B testing': 'seagreen',
+    Surveys: 'salmon',
 }
 
-function OnboardingNotCompletedButton({
-    url,
-    getStartedActionOverride,
-}: {
-    url: string
-    productKey: ProductKey
-    getStartedActionOverride?: () => void
-}): JSX.Element {
-    return (
-        <LemonButton
-            type="primary"
-            onClick={() => {
-                if (getStartedActionOverride) {
-                    getStartedActionOverride()
-                }
-                router.actions.push(url)
-            }}
-        >
-            Get started
-        </LemonButton>
-    )
-}
-
-export function getProductIcon(iconKey?: string | null, className?: string): JSX.Element {
-    return Icons[iconKey || 'IconLogomark']({ className })
+export function getProductIcon(productName: string, iconKey?: string | null, className?: string): JSX.Element {
+    return Icons[iconKey || 'IconLogomark']({
+        className,
+        color: productName ? ProductNameToColor[productName] : 'black',
```

Comment:
does `black` transform for dark mode okay?

Reply:
Turns out I don't need to do this as looking up a productName that doesn't exist will return `undefined` and default the color to white.